.travis.yml

```diff
@@ -8,11 +8,19 @@ services:
 before_install:
   - docker run -d -p 5000:5000 --restart=always --name registry registry:2
 
-jdk:
-  - oraclejdk8
+jdk: oraclejdk8
 
-script:
-  - ./gradlew build -s
+script: ./gradlew build -s
+
+jobs:
+  include:
+    - stage: release
+      script: skip
+      deploy:
+        provider: script
+        script: ./gradlew assemble bintrayUpload
```

Comment:
@bmuschko `script: skip` in line 18 will prevent the default script execution thus I use `assemble` to avoid testes execution.

> Here we should also publish the Javadocs.

How to do that?